packages/js/src/general/routes/alert-center.js

```diff
@@ -18,6 +19,18 @@ export const AlertCenter = () => {
 	const academyLink = useSelectGeneralPage( "selectLink", [], "https://yoa.st/3t6" );
 	const { isPromotionActive } = useSelect( STORE_NAME );
 
+	/**
+	 * Removes the first time configuration notice when the the last step has been completed.
+	 */
+	useEffect( () => {
+		if ( window.wpseoFirstTimeConfigurationData.finishedSteps.includes( "personalPreferences" ) )  {
+			const firstTimeConfigurationNotice = document.getElementById( "yoast-first-time-configuration-notice" );
+			if ( firstTimeConfigurationNotice ) {
+				firstTimeConfigurationNotice.remove();
+			}
+		}
+	}, [] );
+
```

Comment:
While this might work. I think this is a hack upon a hack.
The actual notices are transformed in [App](https://github.com/Yoast/wordpress-seo/blob/6fe61e5b5a682b8885330e82b0e67007f45c9cb9/packages/js/src/general/app.js#L71-L74)

Perhaps you should modify that memoized state instead of removing whatever is rendered. Modifying DOM that is rendered by React is generally not a thing you want to be doing. What if React decides to check and just puts it back?

Reply:
That was my first implementation, the current one is the one we discussed together 🙂